main.tf

```diff
@@ -186,6 +186,23 @@ resource "aws_s3_bucket" "default" {
             replica_kms_key_id = try(rules.value.destination.replica_kms_key_id, null)
             account_id         = try(rules.value.destination.account_id, null)
 
+            # https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-walkthrough-5.html
```

Comment:
Is there a general documentation page we can link or just a walkthrough?

Reply:
this is best documenting the schema